test/verifiedTests/visualization/testVisualizeEpistasis/testFindSubsystemOfGenes.m

```diff
@@ -0,0 +1,48 @@
+% % The COBRAToolbox: testFindSubsystemOfGenes.m
+% %
+% % Purpose:
+% %     - testFindSubsystemOfGenes tests the whether findSubystemOfGenes 
+% %     is working correctly
+% %
+% % Author:
+% %     - Original file: Chintan J Joshi - 11/29/2018
+% 
+% save the current path
+currentDir = pwd;
+
+% initialize the test
+fileDir = fileparts(which('testFindSubsystemOfGenes'));
+cd(fileDir);
+
+% get the inputs
+load('testInputsEpistasisVisualization.mat', 'Coli','genesColi');
```

Comment:
Could you use `Coli = readCbModel('testInputsEpistasisVisualization.mat','modelName','Coli')` to load the model structure? This way, if the model structure ever gets changed in a way that is incompatible with your functions, it would show up on the CI.

Reply:
Just changed that line. and disabled using load for reading the Coli model.